projects/agc-recast.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Recasting of IRIS-HEP Analysis Grand Challenge
```

Comment:
@cranmer @lukasheinrich Throwing up a quick draft version now. Happy to revise and clarify any text.